packages/nimble-components/src/table-column/icon/cell-view/index.ts

```diff
@@ -27,19 +31,28 @@ export class TableColumnIconCellView
     TableColumnEnumCellRecord,
     TableColumnEnumColumnConfig
     >
-    implements IconView {
+    implements IconView, SpinnerView {
     @observable
     public severity: IconSeverity;
 
     @observable
     public text?: string;
 
     @observable
-    public iconTemplate?: ViewTemplate<IconView>;
+    public visualizationTemplate?:
+    | ViewTemplate<IconView>
+    | ViewTemplate<SpinnerView>;
 
     @observable
     public visual?: 'spinner' | 'icon';
```

Comment:
Is visual still useful? can just check if `visualizationTemplate` is defined in the template where this was used?
Think it applies to cell-view and group-header-view